quickwit-serve/src/lib.rs

```diff
@@ -48,28 +45,6 @@ use crate::grpc_adapter::cluster_adapter::GrpcClusterAdapter;
 use crate::grpc_adapter::search_adapter::GrpcSearchAdapter;
 use crate::rest::start_rest_service;
 
-fn display_help_message(rest_socket_addr: SocketAddr) -> anyhow::Result<()> {
```

Comment:
I think that was actually useful.

Reply:
Sounds overkill and will be flooded by other log messages. I'm ok to show something in info like "Server started on".